electrum/commands.py

```diff
@@ -276,6 +275,27 @@ async def restore(self, text, passphrase=None, password=None, encrypt_file=True,
             'msg': d['msg'],
         }
 
+    @command('n')
+    async def restore_sync(self, text, passphrase=None, password=None, encrypt_file=True, wallet_path=None):
```

Comment:
I see the TODO says "create a separate command", but now thinking about it, adding an option to the existing `restore` command seems nicer. e.g. `sync_history=False`. You can add some description for such an option in `command_options` (around line 1300).

Reply:
I have modified the commit to make it an option that polls `wallet.is_up_to_date()` using the wallet object returned by `restore_wallet_from_text()`. I have tested it and it works as intended now.

Reply:
@SomberNight can you verify if the contents of the new commits are acceptable for merge?